python-checks/src/test/resources/checks/incompatibleOperands/comparison.py

```diff
@@ -2,7 +2,7 @@ def builtin_noncompliant(p):
   1 < "1"  # Noncompliant {{Fix this invalid "<" operation between incompatible types (int and str).}}
 #   ^
   complex(1) < complex(1)  # Noncompliant
-  [1] < (1,)  # Noncompliant
+  [1] < (1,)  # FN Noncompliant
```

Comment:
The problem here seems to be a bit different, as for some reason `list` and `tuple` are considered compatible with regard to the `__gt__` method of `tuple`, not sure how though...